apps/src/lab2/levelEditors/types.ts

```diff
@@ -0,0 +1,15 @@
+export interface LevelPredictSettings {
+  is_predict_level: boolean;
```

Comment:
yeah good point, I used snake case because that's what everything on the backend uses, but I see we are using camel case for the ai chat settings, and it was additional work to camelize them when we send them back to the frontend later. I'll update this.